asdf/tests/helpers.py

```diff
@@ -436,7 +442,29 @@ def _assert_extension_type_correctness(extension, extension_type, resolver):
         extension_type.__name__
     )
 
-    for check_type in extension_type.versioned_siblings + [extension_type]:
+    # check the default version
+    types_to_check = [
+        extension_type,
```

Comment:
```suggestion
        extension_type
```
`black` should make this less ugly now.

Reply:
Thanks! I made this change and reran black in 472b0b7